src/components/selectors/permission-filter.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { TextField } from "@mui/material";
+import { Selector, FormControlWrapper } from "../../styled/layouts/admin-layout";
+import strings from "localization/strings";
+
+/**
+ * Disable second selector by default and if first selector is selected, enable second selector
+*/
+const PermissionFilter: React.FC = () => {
+  const [ firstSelector, setFirstSelector ] = React.useState<string>("");
+  const [ secondSelector, setSecondSelector ] = React.useState<string>("");
+
+  /**
+   * We need to disable second selector if first selector is selected
+   * We read that the first selector has "" as value, so we need to check if it is empty
+   * @param event - event object
+   */
+  const handleFirstSelectorChange = (event: React.ChangeEvent<{ value: string }>) => {
+    setFirstSelector(event.target.value);
+    setSecondSelector("");
+  };
+
+  /**
+   * We enable second selector if first selector is selected
+   * @param event - event object
+   */
+  const handleSecondSelectorChange = (event: React.ChangeEvent<{ value: string }>) => {
+    setSecondSelector(event.target.value);
+  };
+
+  return (
+    <FormControlWrapper>
+      <Selector
```

Comment:
These could also be renamed, maybe FilterSelector is more describing.

Reply:
Changed the names now, so both have the word Filter in them as well